paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -157,6 +158,20 @@ internal class PaymentSheetViewModel @Inject internal constructor(
 
     private var deferredIntentConfirmationType: DeferredIntentConfirmationType? = null
 
+    @VisibleForTesting
```

Comment:
Is the visible for testing needed?

Reply:
Not anymore no, I initially made this `internal` before deciding to test the google pay button logic through the `walletsState` flow. Will remove